examples/basic.tsx

```diff
@@ -125,6 +129,10 @@ export default () => {
           <h3>Keyboard navigation (Tab key) disabled</h3>
           <Picker<Moment> {...sharedProps} locale={enUS} tabIndex={-1} />
         </div>
+        <div style={{ margin: '0 8px' }}>
+          <h3>Keyboard event with disabled key</h3>
```

Comment:
It shouldn't be `'Keyboard event with disabled key'`?

Reply:
You also need change title of PR?

Reply:
should i change to ```'Keyboard event with prevent default behaviors'```?

Reply:
OK